public/styles/App.scss

```diff
@@ -1,15 +1,9 @@
-$color: red;
-
-@mixin my-border($color) {
-  border: 1px solid $color;
+.general__nav {
```

Comment:
this isn't following BEM

Reply:
Lets import component styles into this file. Each component should have its own stylesheet